packages/pages/src/common/src/template/types.ts

```diff
@@ -1,5 +1,6 @@
 import { ProjectStructureConfig } from "../project/structure.js";
 import { HeadConfig } from "./head.js";
+import React from "react";
```

Comment:
What this necessary? It's a .ts file that doesn't use react.

Reply:
Ah you're right. I assumed we needed it for React.JSX.Element, but we don't.

Reply:
Oh nvm, we do or else I get an error 
`│ ERROR: Internal Error: Unable to follow symbol for "React"`
from the api-extractor

Reply:
Hmm. This is dubious. 
1. React isn't used in this file at all
2. The code is correct (compiles np) so why does api-extractor need it?

Reply:
Hmmm unfortunately the error doesn't give a whole lot of detail. 
Full error is: 
`ERROR: Internal Error: Unable to follow symbol for "React"
│ You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.`

Someone else had this [error pop-up due to a missing import](https://github.com/microsoft/rushstack/issues/3612) which caused me to try and import React which fixed the error.